gitmine/commands/get.py

```diff
@@ -91,7 +107,50 @@ def __init__(
         )
 
 
-def get_prs(ctx: click.Context, headers: Mapping[str, str]) -> List[Mapping[str, Any]]:
+class Repository:
+    """ Container class for a Github Repository
+    """
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.issues: List[Issue] = []
+        self.prs: List[PullRequest] = []
+
+    def add_issue(self, issue: Issue) -> None:
+        self.issues.append(issue)
```

Comment:
This might be a good spot to practice defensive programming? I guess MyPy takes care of that though...